mplc/experiment.py

```diff
@@ -0,0 +1,138 @@
+# -*- coding: utf-8 -*-
+"""
+An Experiment regroups multiple scenarios and enables to run them and analyze their results.
+"""
+
+import datetime
+import os
+from pathlib import Path
+
+import matplotlib.pyplot as plt
+from loguru import logger
+
+from . import scenario as scenario_module
+from . import utils, constants
+
+DEFAULT_CONFIG_FILE = "../config.yml"
+
+
+class Experiment:
+    def __init__(
+            self,
+            experiment_name=None,
+            nb_repeats=1,
+            scenarios_list=[]
+    ):
```

Comment:
Maybe I can be great to allow one to pass the path as a parameter ? Maybe via **kwargs ?

Reply:
Yes, good idea